doc/nrf/releases_and_maturity/releases/release-notes-2.9.0-nrf54h20.rst

```diff
@@ -0,0 +1,399 @@
+:orphan:
+
+.. _ncs_release_notes_290-nrf54h20:
+
+|NCS| v2.9.0-nRF54H20 Release Notes
+###################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+The |NCS| v2.9.0-nRF54H20 is a nRF54H20-exclusive release, supporting products based on the nRF54H20 SoC.
+
```

Comment:
Add known issues

Reply:
https://nordicsemi.atlassian.net/browse/NCSDK-31013 should be added.

Reply:
https://nordicsemi.atlassian.net/browse/NCSDK-31038

Reply:
@shanthanordic Please check if known issues descriptions are complete